code/game/objects/items/crayons.dm

```diff
@@ -286,6 +315,7 @@
 		target = target.loc
 
 	if(!isValidSurface(target))
+		balloon_alert(user, "can't use there!")
```

Comment:
would it make sense for the balloon alert to show over the target instead of the user?